### ISSUE TITLE
Delete cluster, does not recreate properly

### ISSUE THREAD

**Issue by twalk4821** [CrunchyData/postgres-operator#148]
When I deploy a pod and run show, I get output like this:
```
cluster : timescale-db (centos7-10.1-1.7.1)
├── pod : timescale-db-765bfb667f-7qzhn (Running on gke-db-cluster-default-pool-6fc15428-9nrw) (1/1)
        ├── pvc : timescale-db-pvc
├── deployment : timescale-db
├── service : master-service (10.31.249.74)
└── service : timescale-db (10.31.252.5)
```

but after deleting using the selector method like:
```
pgo delete cluster --selector=timescale --delete-data --delete-backups
```

And creating a cluster again of the same name, I get this:
```
cluster : timescale-db (centos7-10.1-1.7.1)
└── service : master-service (10.31.249.74)
```

Why does the cluster not produce new pods?

**Comment by twalk4821** [comment]
Theory: creation of the load-balancer service on the cluster precludes fully deleting the cluster somehow

**Comment by jmccormick2001** [comment]
fyi, I think I see what could be causing this, it requires a fix in the operator backend...the fix will be included into the 2.6-rc2 release which I plan on making tonight.

**Comment by jmccormick2001** [comment]
fix is merged into pre27, will land in next release.
